I will assume that bridges can be between 1,000 feet and 4,000 feet. Any lenghts outside will be flagged.

In [1]:
import pandas as pd

df = pd.read_csv("bridges.csv")

In [2]:
display(df.describe())

,location,erected,length,lanes
count,107.000000,108.000000,81.000000,92.000000
mean,25.978505,1905.314815,1567.469136,2.630435
std,13.665871,37.173664,747.491523,1.164657
min,1.000000,1818.000000,804.000000,1.000000
25%,15.500000,1884.000000,1000.000000,2.000000
50%,27.000000,1903.000000,1300.000000,2.000000
75%,37.500000,1928.000000,2000.000000,4.000000
max,52.000000,1986.000000,4558.000000,6.000000


In [3]:
display(df[df["length"] < 1000])

,Id,river,location,erected,purpose,length,lanes,clear-g,t-or-d,material,span,rel-l,type
5,E7,A,27.0,1840,HIGHWAY,990.0,2.0,N,THROUGH,WOOD,MEDIUM,S,WOOD
56,E53,A,28.0,1904,RR,965.0,4.0,G,THROUGH,STEEL,MEDIUM,S-F,SIMPLE-T
70,E64,A,29.0,1923,HIGHWAY,885.0,4.0,G,THROUGH,STEEL,MEDIUM,F,ARCH
72,E70,A,27.0,1926,HIGHWAY,860.0,4.0,G,THROUGH,STEEL,MEDIUM,S-F,SUSPEN
73,E69,A,26.0,1926,HIGHWAY,884.0,4.0,G,THROUGH,STEEL,MEDIUM,S-F,SUSPEN
80,E71,A,25.0,1928,HIGHWAY,860.0,4.0,G,THROUGH,STEEL,MEDIUM,S-F,SUSPEN
88,E107N,A,39.7,1945,RR,840.0,2.0,G,THROUGH,STEEL,MEDIUM,S-F,SIMPLE-T
94,E98,M,22.0,1951,HIGHWAY,900.0,4.0,G,THROUGH,STEEL,MEDIUM,F,CONT-T
98,E82,O,42.0,1955,HIGHWAY,804.0,NaN,G,THROUGH,STEEL,NaN,F,SIMPLE-T
101,E86,A,33.0,1961,HIGHWAY,980.0,4.0,G,DECK,STEEL,MEDIUM,S-F,CONT-T


In [4]:
display(df[df["length"] > 4000])

,Id,river,location,erected,purpose,length,lanes,clear-g,t-or-d,material,span,rel-l,type
31,E34,O,41.0,1888,RR,4558.0,2.0,G,THROUGH,STEEL,LONG,F,SIMPLE-T


I found that 12 bridges are below my 1,000 feet range assumption. I think that the shorter bridges are not impossible. While they fall below my assumption most of them are reasonable and do not fall too far below. The only one I would flag is the row 5, as it is listed as 990 in length but made out of wood. I would flag this for verification as the rest are marked as steel.
I would flag the longer bridge that is over 4000 feet as it seems to be an outlier and the only one of this length recorded in the data. My action would be to keep both unchanged and flag them for verification. I feel that there is not enough evidence in the data to remove them.